lsp-svelte.el

```diff
@@ -0,0 +1,53 @@
+;;; lsp-svelte.el --- LSP Svelte integration -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 Stepan Lusnikov
+
+;; Author: Stepan Lusnikov <endenwer@gmail.com>
```

Comment:
I'm not sure if this should be `lsp maintainers` or your name.
Maybe @yyoncho can confirm that

Reply:
I checked different files and they have different authors. For example, one of the last pr #1832

Reply:
It is fine to stay as it is.